cloud/azure/azure.md

```diff
@@ -3,7 +3,9 @@ layout: page
 title: "Portworx on Azure"
 keywords: portworx, azure, PaaS, IaaS, docker, converged, cloud
 sidebar: home_sidebar
-redirect_from: "/portworx-on-azure.html"
+redirect_from:
+  - /portworx-on-azure.html
+  - /cloud/azure/azure.html
```

Comment:
Here is the problem @GranSpencer09 - you're overwriting it's own page with a redirect to itself. This change is an issue.